### PR TITLE
Embed images

### PR DESCRIPTION
**Pull request by Bartzi**
allows embedding of images in a document as requested in #178 
sneak preview:

image

### CODE REVIEW

**Comment by Bartzi** [comment]
alright I've addressed all comments ;) please review again

File: _1327/documents/templates/attachments.html [hunk 1]
```diff
@@ -25,6 +25,9 @@
 				<th>{% trans "Display name" %}</th>
 				<th>{% trans "Filename" %}</th>
 				<th>{% trans "Created" %}</th>
+				<th data-toggle="tooltip" data-placement="top" title="{% trans "Untick if attachment is only used as embedded image" %}">
+                    {% trans "Downloadable" %} <span class="label label-info">?</span>
+                </th>
```
Comment (Nef10) [inline _1327/documents/templates/attachments.html]: use tabs instead of spaces

File: _1327/documents/templates/editor.html [hunk 2]
```diff
@@ -36,14 +60,83 @@
...
+				[{
+					name: "ImageGroup",
+					data: [{
+						name: "cmdAddImage",
+						toggle: false,
+						title: "Add Image",
+						icon: "glyphicon glyphicon-picture",
+						callback: function (e) {
+							// get all attachments
+							$.get(
```
Comment (invliD) [inline _1327/documents/templates/editor.html]: Please use `$.getJSON` instead of `$.get` and `JSON.parse`.

**Comment by Nef10** [comment]
In the fullscreen markdown editor the select image dialog appears in the background

**Comment by Nef10** [comment]
On the upload attachment site it would be nice to extent the "Can be downloaded by users" Checkbox description. E.g. "Can be downloaded by users, if not checked it can only be used as embedded image" or "Do not show as attachment (for embedded images)"

**Comment by Nef10** [comment]
:+1:

**Comment by invliD** [comment]
code lgtm, please squash all commits.

**Comment by invliD** [comment]
:+1:

**Pull request merged**